README.md

```diff
@@ -234,47 +234,33 @@ class Blog < ActiveRecord::Base
   has_many :recent_posts, -> { order(published_at: desc).limit(5) }, class_name: 'Post'
 end
 ```
-This can be reworked to push the order/limit into a database view:
+This can be reworked to push the order/limit into lateral joins like this:
```

Comment:
Can you update the "... pushing the problematic SQL into the database via database views" text a few lines up to say "... pushing the problematic SQL into the database via lateral joins (or database views if your database doesn't support lateral joins)"

Reply:
Hi @jturkel. I updated it, and after reading it, I'm wondering if we should leave the view example. After all, as you mentioned, this won't work for all databases.

Reply:
I think it's fine to leave it out since most modern databases support lateral joins.